Sources/FaunaDB/Value.swift

```diff
@@ -331,45 +369,58 @@ extension BytesV: Equatable {
     }
 }
 
+/// Represents a query value in the FaunaDB query language.
+/// [Reference](https://fauna.com/documentation/queries#values-special_types).
+public struct QueryV: Value, AsJson {
+
+    internal let value: [String: Any]
```

Comment:
Also, `fileprivate` would be more restrict if the value is not used no where else.